api/v1beta1/openstackdataplaneservice_types.go

```diff
@@ -90,6 +96,10 @@ type OpenStackDataPlaneServiceSpec struct {
 	// ContainerImages struct field names from
 	// github.com/openstack-k8s-operators/openstack-operator/apis/core/v1beta1
 	ContainerImageFields []string `json:"containerImageFields,omitempty" yaml:"containerImageFields,omitempty"`
+
+	// EDPMServiceName - name to use for edpm_service_name ansible variable
+	// +kubebuilder:validation:Optional
+	EDPMServiceName string `json:"edpmServiceName,omitempty" yaml:"edpmServiceName,omitempty"`
```

Comment:
shouldn't it be:
```suggestion
	DataplaneServiceName string `json:"edpmServiceName,omitempty" yaml:"dataplaneServiceName,omitempty"`
```

Reply:
well, it already is a DataPlaneService, but with a different name, so I thought calling this field something different (and aligned with the ansible variable) would make sense. But, I'm happy to change it, let's see if there's any other input.

Reply:
if it is aligned with the ansible variable, then it is good (it was never a blocker to me, but I wonder when we should use `dataplane` or `EDPM`  for naming things)